src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -3044,6 +3061,7 @@ namespace AirflowNetworkBalanceManager {
         // Assign external node azimuth, should consider combining this with the above to avoid the repeated search
         for (int i = 1; i <= dataAirflowNetworkBalanceManager.AirflowNetworkNumOfExtNode; ++i) {
             for (j = 1; j <= AirflowNetworkNumOfSurfaces; ++j) {
+                if (MultizoneSurfaceData(j).SurfNum == 0) continue;
```

Comment:
If an invalid surface is specified, a severe error is issued above, but other input validation continues here, so need to protect against `SurfNum=0`.

Reply:
I think it would be better to make the error check fatal in the previous error check section:

![Capture2](https://user-images.githubusercontent.com/5970656/93224069-bbdbd200-f73e-11ea-863a-0400d4215d28.PNG)

by inserting:

                ShowFatalError(RoutineName + "Errors found getting inputs. Previous error(s) cause program termination.");